src/components/postDetail/reservationPanel/ReservationPanel/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { useEffect, useState } from 'react';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import classNames from 'classnames/bind';
+import { FormProvider, useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { ERROR_MESSAGE } from '@/constants';
+
+import { BaseButton, CountButton } from '@/components/commons/buttons';
+import { FormDropdown } from '@/components/commons/inputs/FormDropdown';
+import { ConfirmModal, ModalButton } from '@/components/commons/modals';
+import Calendar from '@/components/postDetail/reservationPanel/Calendar';
+import useToggleButton from '@/hooks/useToggleButton';
+
+import styles from './ReservationPanel.module.scss';
+
+const cx = classNames.bind(styles);
+const { minPlayMember, availableSchedule } = ERROR_MESSAGE;
+
+type ReservationPanelProps = {
+  activityId: number;
+  maxCount: number;
+};
+
+type AvailableTimesOptions = {
+  title: string;
+  value: number;
+};
+
+const ReservationPanel = ({ activityId, maxCount }: ReservationPanelProps) => {
+  const ReservationSchema = z.object({
+    headCount: z.number().min(1, { message: minPlayMember.min }),
+    scheduleId: z.number().refine((id) => id !== 0, { message: availableSchedule.min }),
+  });
+
```

Comment:
```suggestion

  const ReservationSchema = z.object({
    headCount: z.number().min(1, { message: minPlayMember.min }),
    scheduleId: z.number().refine((id) => id !== 0, { message: availableSchedule.min }),
  });

const ReservationPanel = ({ activityId, maxCount }: ReservationPanelProps) => {

```
ReservationSchema 는 컴포넌트 밖에 선언해 줘도 좋을 거 같아요

Reply:
네! 반영했습니다~